src/pressedAnotherOperator.js

```diff
@@ -0,0 +1,26 @@
+const setCharAt = (str, index, chr) => {
+    if (index > str.length - 1) return str;
+    return str.substring(0, index) + chr + str.substring(index + 1);
+  };
+
+  const pressedAnotherOperator = (result, button) => {
```

Comment:
@grimxreaper is your formatting on save enabled? The indentation here looks off.

Reply:
Yes it is enabled but that does look off, I will change that in the next commit if that's okay. 👍🏼

Reply:
Sounds good